panel/src/components/Dialogs/FileRenameDialog.vue

```diff
@@ -84,10 +84,7 @@ export default {
         .then(file => {
 
           // move form changes
-          this.$store.dispatch("content/move", [
-            this.$store.getters["content/id"](this.file.id),
-            this.$store.getters["content/id"](file.id)
-          ]);
+          this.$store.dispatch("content/move", [this.file.id, file.id]);
```

Comment:
I might have to double check this. I assumed since it was implemented with just `this.file.id` so far, it should be kept that way. But maybe it is a bug in the current release already?

Reply:
I thought so as well. But I have to confess that I didn't test it yet.